system-int-test/test/cmr/system_int_test/search/granule_geojson_search_test.clj

```diff
@@ -0,0 +1,155 @@
+        across-am-poly (make-gran "across-am-poly" (polygon 170 -10, -175 -10, -170 10, 175 10, 170 -10))
+        on-np (make-gran "on-np" (polygon 45 85, 135 85, -135 85, -45 85, 45 85))
+        on-sp (make-gran "on-sp" (polygon -45 -85, -135 -85, 135 -85, 45 -85, -45 -85))
+
+        ;; polygon with holes
+        outer (umm-s/ords->ring -5.26,-2.59, 11.56,-2.77, 10.47,8.71, -5.86,8.63, -5.26,-2.59)
+        hole1 (umm-s/ords->ring 6.95,2.05, 2.98,2.06, 3.92,-0.08, 6.95,2.05)
+        hole2 (umm-s/ords->ring 5.18,6.92, -1.79,7.01, -2.65,5, 4.29,5.05, 5.18,6.92)
+        polygon-with-holes (make-gran "polygon-with-holes" (poly/polygon [outer hole1 hole2]))
+
+        ;; Cartesian Polygons
+        wide-south-cart (make-cart-gran "wide-south-cart" (polygon -70 -30, 70 -30, 70 -20, -70 -20, -70 -30))
+        very-wide-cart (make-cart-gran "very-wide-cart" (polygon -180 40, -180 35, 180 35, 180 40, -180 40))
+        very-tall-cart (make-cart-gran "very-tall-cart" (polygon -160 90, -160 -90, -150 -90, -150 90, -160 90))
+
+        outer-cart (umm-s/ords->ring -5.26 -22.59 11.56 -22.77 10.47 -11.29 -5.86 -11.37 -5.26 -22.59)
+        hole1-cart (umm-s/ords->ring 6.95 -17.95 2.98 -17.94 3.92 -20.08 6.95 -17.95)
+        hole2-cart (umm-s/ords->ring 5.18 -13.08 -1.79 -12.99 -2.65 -15 4.29 -14.95 5.18 -13.08)
+        polygon-with-holes-cart (make-cart-gran "polygon-with-holes-cart" (poly/polygon [outer-cart hole1-cart hole2-cart]))
+
+        ;; Points
+        washington-dc (make-gran "washington-dc" (p/point -77 38.9))
+        richmond (make-gran "richmond" (p/point -77.4 37.54))
+        north-pole (make-gran "north-pole" (p/point 0 90))
+        south-pole (make-gran "south-pole" (p/point 0 -90))
+        am-point (make-gran "am-point" (p/point 180 22))]
+    (index/wait-until-indexed)
+
+    (testing "GeoJSON Failure cases"
+      (are3 [shapefile additional-params regex]
+        (is (re-find regex
+                     (first (:errors (search/find-refs-with-multi-part-form-post
+                                       :granule
+                                       (let [params [{:name "shapefile"
+                                                      :content (io/file (io/resource (str "shapefiles/" shapefile)))
+                                                      :mime-type "application/geo+json"}]]
+                                          (if (seq additional-params)
+                                            (conj params additional-params)
+                                            params)))))))
+                                  
+        "All granules query"
+        "polygon_with_hole.geojson" {} #"The CMR does not allow querying across granules in all collections with a spatial condition"
+
+        "Features with no geometry"
+        "single_point_dc_BAD.geojson" {:name "provider" :content "PROV1"} #"Failed to parse GeoJSON file"))
```

Comment:
Maybe change to "Failed to parse GeoJSON shapefile"?

Reply:
Done.

Reply:
Done.